tasks/discover-granules/tests/index.js

```diff
@@ -119,57 +118,143 @@ test('discover granules using HTTP', async (t) => {
   await validateConfig(t, event.config);
 
   try {
-    await assertDiscoveredGranules(t, await discoverGranules(event));
+    assertDiscoveredGranules(t, await discoverGranules(event));
```

Comment:
`await assertDiscoveredGranules`  There are a couple of other places missing `await`

Reply:
Ugh! I can't believe I missed those.